docker/default.yml

```diff
@@ -8,6 +8,7 @@ services:
   uwsgi:
     image: intelowlproject/intelowl:${REACT_APP_INTELOWL_VERSION}
     container_name: intelowl_uwsgi
+    hostname: uwsgi
```

Comment:
is this really needed?

Reply:
We had some with the name, some without. Just to have the same syntax everywhere